src/main/java/in/twizmwaz/cardinal/module/modules/scoreboard/ScoreboardModule.java

```diff
@@ -478,36 +471,41 @@ public void renderTeamTitle(TeamModule teamModule) {
     public void renderTeamScore(ScoreModule score) {
         TeamModule teamModule = score.getTeam();
         Team team = scoreboard.getTeam(teamModule.getId() + "-s");
-        team.setPrefix(teamModule.getColor() + StringUtils.trimTo(teamModule.getName(), 0, 14));
-        team.setSuffix(StringUtils.trimTo(teamModule.getName(), 14, 30));
+        String fullName = score.getScore() + " " + teamModule.getCompleteName();
+        team.setPrefix(StringUtils.trimTo(fullName, 0, 16));
+        team.setSuffix(StringUtils.trimTo(fullName, 16, 32));
         if (team.getEntries().size() > 0) {
-            setScore(objective, new ArrayList<>(team.getEntries()).get(0), score.getScore());
+            setScore(objective, new ArrayList<>(team.getEntries()).get(0), currentScore);
         } else {
-            String name = teamModule.getColor() + "";
+            String name = ChatColor.RESET + "";
             while (used.contains(name)) {
-                name = teamModule.getColor() + name;
+                name = ChatColor.RESET + name;
             }
             team.addEntry(name);
-            setScore(objective, name, score.getScore());
+            setScore(objective, name, currentScore);
             used.add(name);
         }
+        currentScore++;
     }
 
     public void renderTeamBlitz(TeamModule teamModule) {
         Team team = scoreboard.getTeam(teamModule.getId() + "-b");
-        team.setPrefix(teamModule.getColor() + StringUtils.trimTo(teamModule.getName(), 0, 14));
-        team.setSuffix(StringUtils.trimTo(teamModule.getName(), 14, 30));
+        String fullName = teamModule.size() + " " + teamModule.getCompleteName();
+        team.setPrefix(StringUtils.trimTo(fullName, 0, 16));
+        team.setSuffix(StringUtils.trimTo(fullName, 16, 32));
         if (team.getEntries().size() > 0) {
-            setScore(objective, new ArrayList<>(team.getEntries()).get(0), teamModule.size());
+            String entry = new ArrayList<>(team.getEntries()).get(0);
+            setScore(objective, entry, scoreboard.getObjective("scoreboard").getScore(entry).getScore());
         } else {
-            String name = teamModule.getColor() + "";
+            String name = ChatColor.RESET + "";
```

Comment:
This should be kept as teamModule.getColor() as it will usually be in the middle of the team name, not the beginning of the entry.